libraries/api/include/golos/api/discussion_helper.hpp

```diff
@@ -11,13 +12,29 @@ namespace golos { namespace api {
 
     comment_metadata get_metadata(const comment_api_object &c);
 
+    struct get_comment_content_res {
+        std::string title;
+        std::string body;
+        std::string json_metadata;
+    };
```

Comment:
Remove this structure, you should use fill_comment_content. No additional structures should be used.